core/vm/instructions.go

```diff
@@ -908,6 +918,25 @@ func makeLog(size int) executionFunc {
 	}
 }
 
+func opPaygas(pc *uint64, interpreter *EVMInterpreter, contract *Contract, memory *Memory, stack *Stack) ([]byte, error) {
```

Comment:
In the reference implementation, they used callContext, why we don't have it here? https://github.com/quilt/go-ethereum/blob/9c4f387bc356af0e75019764d5730e5ea8b36d7f/core/vm/instructions.go#L914

Reply:
because call context seems to be introduced in more recent version ofthe vm than harmony uses currently. but symantically we do exactly the same: call context is a struct with stack, contract and memory, and we pass exactly the same values just as separate arguments.